### PR TITLE
Added general intro and changes for accessibility.

### PR DESCRIPTION
**Pull request by Astro-Choco**
Related JIRA ticket: DEVX-704

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -44,7 +65,7 @@ To be transparent and set expectations right, we've applied a Beta label on any
 In the meantime, we will work hard to improve the Zettle Developer Platform. An example of the ongoing work is to improve the API documentation.
```
Comment (AndreaFilyo) [inline CHANGELOG.adoc:65]: Cannot figure out if this is displayed anywhere, but I would suggest the following wording:
"We are currently working to improve the Zettle Developer Platform, including the API documentation."

**Review by AndreaFilyo — DISMISSED** [review]
Looks good to me, only one minor comment.

File: CHANGELOG.adoc (hunk 1 above)
Comment (Astro-Choco) [reply CHANGELOG.adoc]: Thanks @AndreaFilyo . I fixed your comment. :)

**Review by taizamontenegro — APPROVED** [review]
Very clear description 👏🏽🎉

**Pull request merged**